website/util/__init__.py

```diff
@@ -69,7 +70,7 @@ def api_url_for(view_name, _absolute=False, _xml=False, *args, **kwargs):
 def api_v2_url(path_str,
                params=None,
                base_route=website_settings.API_DOMAIN,
-               base_prefix='',
+               base_prefix=api_settings.API_BASE,
```

Comment:
There may not be an advantage--I only thought of it because we've had to do it before. Like I said, just something to consider. If you decide it's not advantageous, keep it as is. Follow your :heart: